force-app/main/default/lwc/.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "extends": ["@salesforce/eslint-config-lwc/recommended"],
+    "extends": ["@salesforce/eslint-config-lwc/recommended", "prettier"],
```

Comment:
Why is `prettier` for ESLint in here?

Reply:
Not entirely sure about the benefit but it is supposed to help to better integrate ESLint with Prettier.
@ntotten?

Reply:
I know what it does. My question is, and that should become tribal knowledge for the team @pozil, what conflicting rules are there to make this change relevant.

Reply:
Ok so just did the test to remove that. The conflict with Prettier is pretty simple, we get loads of these errors:
> Unquoted property 'handleContactSelect' found  quote-props

It would require to quote all function names in our Controllers and Helpers.